### DISCUSSION TITLE
Will OpenAPI generation still be supported?

### DISCUSSION THREAD

**Discussion by Judekeyser** [pydantic/pydantic#5763] [Pydantic V2]
Will OpenAPI generation, using `schema` from `pydantic.schema`, still be supported?

I rely on it to infer valid schemas for OpenAPI (with APISpec) and so far it works just great 🙏

**Comment by samuelcolvin** [comment]
See https://github.com/pydantic/pydantic/issues/4666#issuecomment-1298456812 - we're aiming to support JSON Schema 2020-12 which is compatible with OpenAPI 3.1.

**Comment by Judekeyser** [comment]
What kind of feature would require OpenAPI 3.1? I ask because since the version seems to be really on the edge, it might be quite difficult to integrate newly created schemas with previous connectors that are still working under 3.0.2 or 3.0.3.
